### PR TITLE
PROD release may 2024

### PR DESCRIPTION
**Pull request by ricardofabila**
This is the change that adds scheduling@mento.com o the attendees list for the google calendar spy to work

### CODE REVIEW

File: apps/web/pages/auth/login.tsx [hunk 1]
```diff
@@ -39,7 +39,6 @@ interface LoginValues {
   csrfToken: string;
 }
 export default function Login({
-  csrfToken,
```
Comment (kball) [inline apps/web/pages/auth/login.tsx:42]: I know we talked briefly about this but even after digging in I still don't understand why we're removing the csrf token in this PR. That feels like it has further reaching consequences that are unrelated to adding a cc. Can you either explain in more detail why this is necessary or remove it?

**Review by kball — CHANGES_REQUESTED** [review]

File: apps/web/pages/auth/login.tsx (hunk 1 above)
Comment (ricardofabila) [reply apps/web/pages/auth/login.tsx:42]: if you run 'yarn dev' it will break because of the cfsf token, which is not used. it was used in the code for login with password but that is commented out. thi code is live on stage and it works. it's ok. 👍🏼

Comment (kball) [reply apps/web/pages/auth/login.tsx:42]: ah got it, makes sense

**Review by kball — APPROVED** [review]

**Comment by ricardofabila** [comment]
Do not merge until we are ready to release the BE 👍🏼
